Packs/CommonTypes/ReleaseNotes/3_3_96.md

```diff
@@ -0,0 +1,5 @@
+
+#### Indicator Types
+
+- **IPv6**
+Updated the extraction test playbook and the IPv6 regex.
```

Comment:
removed the test playbook change as its not needed in the release notes of the indicator type